docs/reference/modules/discovery/zen.asciidoc

```diff
@@ -56,11 +56,9 @@ serves as a protection against (partial) network failures where node may unjustl
 think that the master has failed. In this case the node will simply hear from
 other nodes about the currently active master.
 
-If `discovery.zen.master_election.filter_client` is `true`, pings from client nodes (nodes where `node.client` is
-`true`, or both `node.data` and `node.master` are `false`) are ignored during master election; the default value is
-`true`. If `discovery.zen.master_election.filter_data` is `true`, pings from non-master-eligible data nodes (nodes
-where `node.data` is `true` and `node.master` is `false`) are ignored during master election; the default value is
-`false`. Pings from master-eligible nodes are always observed during master election.
+If `discovery.zen.master_election.ignore_non_master_pings` is `true`, pings from nodes that are not master
```

Comment:
There are a few negatives here ("non", "ignore") which makes thinking about this setting tricky. I think that this setting will be easier to think about if it is expressed as a positive. So if it is `ignore_non_master_pings` and I want to know if non-master pings are included or not, I have to negate the value of this setting. But if the setting is changed to `include_non_master_pings` then whether or not they are included is equal to the value of the setting.